packages/nuxi/src/cli-wrapper.ts

```diff
@@ -4,6 +4,7 @@
 import { fileURLToPath } from 'node:url'
 import { fork } from 'node:child_process'
 import type { ChildProcess } from 'node:child_process'
+import mri from 'mri'
```

Comment:
Can we do this without importing `mri` in this file? I think the parsing is simple enough for `--inspect`.